.devcontainer/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM julia:1.7
 
-RUN apt-get update && apt-get install -y xz-utils bzip2 sudo git unzip
```

Comment:
This change seems unrelated

Reply:
On second thought, think this solves it a bit more elegantly: https://github.com/JuliaPackaging/Yggdrasil/pull/5541